internal/db/migrations/20250509030046_restore_sip_status_index.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Modify "sip" table
+ALTER TABLE `sip` ADD INDEX `sip_status_idx` (`status`);
```

Comment:
You probably noticed, this was missing from a previous PR, I'll keep it in a separate commit and squash the other two.